### PR TITLE
CSCFAIRMETA: Split Qvain translations to separate files

### PR DESCRIPTION
**Pull request by tahme**

### CODE REVIEW

File: etsin_finder/frontend/locale/english/qvain.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import general from './qvain.general'
+import history from './qvain.history'
+import home from './qvain.home'
+import nav from './qvain.nav'
+import project from './qvain.project'
+import rightsAndLicenses from './qvain.rightsAndLicenses'
+import temporalAndSpatial from './qvain.temporalAndSpatial'
+import validationMessages from './qvain.validationMessages'
+
+const qvainFinnish = {
```
Comment (ismotee) [inline etsin_finder/frontend/locale/english/qvain.js:14]: qvainEnglish?

File: etsin_finder/frontend/locale/english/qvain.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  history,
+  home,
+  nav,
+  project,
+  rightsAndLicenses,
+  temporalAndSpatial,
+  validationMessages,
+}
+
+export default qvainFinnish
```
Comment (ismotee) [inline etsin_finder/frontend/locale/english/qvain.js:117]: qvainEnglish

**Review by ismotee — CHANGES_REQUESTED** [review]

File: etsin_finder/frontend/locale/english/qvain.js (hunk 1 above)
Comment (tahme) [reply etsin_finder/frontend/locale/english/qvain.js]: fixed

File: etsin_finder/frontend/locale/english/qvain.js (hunk 2 above)
Comment (tahme) [reply etsin_finder/frontend/locale/english/qvain.js]: fixed

**Review by ismotee — APPROVED** [review]

**Pull request merged**